modin/core/execution/dask/implementations/pandas_on_dask/io/io.py

```diff
@@ -171,9 +171,6 @@ def to_dask(cls, modin_obj):
         dask.dataframe.DataFrame or dask.dataframe.Series
             Converted object with type depending on input.
         """
-        import_optional_dependency(
-            "dask-expr", "dask-expr is required to create a Dask DataFrame."
-        )
```

Comment:
The default message from dask.dataframe is more user friendly, so I suggest not using this method.

Reply:
![image](https://github.com/modin-project/modin/assets/36244538/58c7c0b4-755a-4aca-9811-ffcffe2f0cbb)

Reply:
I also think that we should not check the version for dusk_expr. Yes, it has problems in 1.0.1 version, but this is a problem in Dusk, not our compatibility with it. In addition, this library releases new versions very quickly and, in my opinion, users are unlikely to encounter this problem in the future.